src/Frontend/src/router/index.ts

```diff
@@ -1,43 +1,56 @@
-import { createRouter, createWebHashHistory, type RouteRecordRaw } from "vue-router";
-import config from "./config";
+import { createRouter, createWebHashHistory, type RouteRecordRaw, RouteRecordSingleViewWithChildren } from "vue-router";
+import config, { RouteItem } from "./config";
 
 function meta(item: { title: string }) {
   return { title: `${item.title} • ServicePulse` };
 }
 
+function addChildren(parent: RouteRecordSingleViewWithChildren, item: RouteItem){
+  if(item.children){
```

Comment:
is prettier turned on? This isn't spaced correctly

Reply:
Fixed